app/scripts/modules/core/src/managed/PreDeploymentStepCard.tsx

```diff
@@ -70,25 +97,27 @@ export const PreDeploymentStepCard = memo(({ step, application, reference }: Pre
     return null;
   }
 
-  const { iconName, appearance, title } = cardConfigurationByType[type];
+  const { iconName, title } = cardConfigurationByType[type];
 
   return (
     <StatusCard
-      appearance={appearance[status]}
+      appearance={cardAppearanceByStatus[status]}
       active={status !== 'NOT_STARTED'}
       iconName={iconName}
       timestamp={getTimestamp(startedAt, completedAt)}
       title={title(step)}
       actions={
         link && (
-          <Button
+          <a
```

Comment:
I did a bad thing on both the `See changes` button and this button — I should've used an `<a/>` but instead used `window.open` because... I'm not sure I was paying enough attention to think through it fully. Corrected both here _and_ for the 'See changes' button.